civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5486,6 +5488,50 @@ function civihr_employee_portal_form_views_exposed_form_alter(&$form, &$form_sta
             }
         }
     }
+
+    if (substr($form['#id'], 0, 33) === 'views-exposed-form-civihr-report-') {
+        foreach ($form as $key => $value) {
+          if (isset($form['#info']["filter-$key"])) {
+            $form[$key]['value'] = generateFilterInputStructure($form['#info']["filter-$key"]['label']);
+          }
+
+          if (isset($form[$key]['min'])) {
+            $form[$key]['min'] = generateFilterInputStructure($form[$key]['min']['#title']);
+          }
+
+          if (isset($form[$key]['max'])) {
+            $form[$key]['max'] = generateFilterInputStructure($form[$key]['max']['#title']);
+          }
+        }
+    }
+
+    $form['submit']['#value'] = 'Filter';
+}
+
+/**
+ * Generate the necessary structure for the filter
+ *
+ * @param  [String] $title
+ * @return [array]
+ */
+function generateFilterInputStructure($title) {
```

Comment:
I don't think the name and the description of the function accurately reflects what the function actually does. Doesn't this function add a calendar button that is bound to the angular datepicker? If yes, then assign a name and function that describes that. Otherwise this is too generic